auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -40,19 +40,23 @@
           </v-tooltip>
 
           <v-form ref="addBusinessForm" lazy-validation class="mt-6">
-            <!-- Business Identifier -->
-            <v-text-field
-              filled req persistent-hint validate-on-blur
-              label="Incorporation Number or Registration Number"
-              hint="Example: BC1234567, CP1234567 or FM1234567"
-              :rules="businessIdentifierRules"
-              v-model="businessIdentifier"
-              @blur="formatBusinessIdentifier()"
-              class="business-identifier mb-n2"
-              aria-label="Incorporation Number and Password or Passcode"
-              autofocus
+            <!-- Business Identifier and Name -->
+            <!-- NB: use v-if to re-mount component between instances -->
+            <BusinessLookup
+              v-if="dialog"
+              @business="businessName = $event.name; businessIdentifier = $event.identifier"
```

Comment:
Can we set the event to one variable called `business` and then we can call the name with `business.name` for example? What do you think Sev?

Reply:
Yes, that would surely work, but I wanted to change as little as possible in this component (since I may have to FF this) and `businessIdentifier` is used in a lot of places in here.

Reply:
Karim, OK if I resolve this conversation?

Reply:
Yes Sev! I was trying to resolve it myself but I have no access to do so 🤣